### DISCUSSION TITLE
Welcome to pandoc Discussions!

### DISCUSSION THREAD

**Discussion by jgm** [jgm/pandoc#9306] [Announcements]
This is the place to ask and answer questions about pandoc!

(The old pandoc-discuss list was frozen after a spam attack in December.  Despite repeated requests, Google has not reenabled it, so we are moving discussion here, at least for now.)

**Comment by jgm** [comment]
There is a searchable archive of the old pandoc-discuss list (2007-2023) here:
https://inbox.vuxu.org/pandoc-discuss/

**Comment by tbdalgaard** [comment]
Have tried to search Github docs for information about subscribing to discussions via E-mail. Could someone please provide details about that either here or at Pandoc.org, Help-section?

**Reply by denismaier** [reply]
You can watch the pandoc repo on the main page. Under "Custom" select which notifications you want:

![grafik](https://github.com/jgm/pandoc/assets/4668198/81793ece-3157-4bef-8a33-644574b6f560)

![grafik](https://github.com/jgm/pandoc/assets/4668198/4c9d772a-f22e-4517-98b8-82a81c0de2fe)
